### DISCUSSION TITLE
thoughts-on-what-is-dotnet-and-why-should-you-choose-it/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#112] [Announcements]
# thoughts-on-what-is-dotnet-and-why-should-you-choose-it/

In this post I discus the first post in Microsoft's 'What is .NET, and why should you choose it?' series and give my thoughts

https://andrewlock.net/thoughts-on-what-is-dotnet-and-why-should-you-choose-it/

**Comment by georghinkel** [comment]
My favorite feature of this post is that it also works as a very comprehensive link collection to detailed technical articles that will introduce you in the details of every aspect of the platform. As someone who has spent a significant part of my career trying to evaluate and assess new technologies and creating concepts of how and why they could solve a given problem, such an article with an overview of mostly everything (e.g., EF Core is hardly mentioned) one needs to know and links to details, beyond any marketing bla-bla, is just invaluable.

**Comment by davidmosborne** [comment]
Great article, as always Andrew. You've used the word 'tenant' a couple of times, when I think you meant to use the word 'tenet'?

**Reply by andrewlock** [reply]
🤦‍♂️ Haha, thanks, fixed (shortly)

**Comment by chrisxfire** [comment]
Andrew, did you learn anything new from this article?  Do you expect to learn anything new from the series?

**Reply by andrewlock** [reply]
There's lots of little bits already mentioned, the main one I learned about was the "mono repo" used for building for linux!

**Comment by mmorsli** [comment]
I think this should be : They're its North Star
instead of:
They're it's North Star.
